### PR TITLE
Update tinyMCE to version 4.3.3

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Version 4.3

See the changes here:
http://blog.tinymce.com/2015/11/24/tinymce-4-3-brings-new-features-stability-improvements/

And the changeling here:
https://github.com/tinymce/tinymce/blob/master/changelog.txt
#### Testing

D/L https://github.com/dgt41/joomla-cms/archive/tinyMCE43.zip
Install joomla as usual and test that tinyMCE didn't introduce any bug

### CODE REVIEW

**Comment by infograf768** [comment]
version change needed in the xml

**Comment by dgt41** [comment]
New lang string "Source code"

**Comment by infograf768** [comment]
>New lang string "Source code"

OK, I guess this is not anyway for 3.5.0, right?

**Comment by dgt41** [comment]
@infograf768 I guess I need to bring support for codesample here as well, but it looks very nice:
screen shot 2015-11-27 at 18 36 06

**Comment by Gerlof** [comment]
I have tested this item :white_check_mark: successfully on a9958710c63dd021ec513f9a492ff2ac89bc7ebb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

File: plugins/editors/tinymce/tinymce.xml [hunk 1]
```diff
@@ -389,6 +389,14 @@
 					<option value="0">JOFF</option>
 				</field>
 
+				<field name="code_sample" type="accesslevel"
+					default="1"
+					description="PLG_TINY_FIELD_CODESAMPLE_DESC"
+					label="PLG_TINY_FIELD_CODESAMPLE_LABEL"
+				>
+					<option value="0">JOFF</option>
```
Comment (wilsonge) [inline plugins/editors/tinymce/tinymce.xml]: I think you need a value for "on" :stuck_out_tongue:

Comment (dgt41) [inline plugins/editors/tinymce/tinymce.xml]: Copy paste gone wrong 😄
But I also forgot the caption and the non editable plugin and also 4.3.1 is released so I will have to update this

**Comment by anibalsanchez** [comment]
Tried to install in 5.6.14... but I am getting error 500 in the final installation steps:

http://..../joomla-cms-tinyMCE43/installation/index.php?task=InstallConfig - 500 Internal Server Error

It may not be even related with the new editor<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on a9958710c63dd021ec513f9a492ff2ac89bc7ebb<br /><br />Test OK, on the latest staging with diff file<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by mikeh000** [comment]
So is this coming to Joomla 3.5?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by brianteeman** [comment]
RTC - thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by dgt41** [comment]
@mikeh000 unfortunately 3.5 is in Beta phase which means new features could not be introduced, so this has to wait till 3.5.1.

**Comment by brianteeman** [comment]
How is updating a library a new feature

**Comment by wilsonge** [comment]
No I will merge this before 3.5.1. Just get that thing updated.

File: plugins/editors/tinymce/tinymce.xml (hunk 1 above)
Comment (dgt41) [inline plugins/editors/tinymce/tinymce.xml]: Actually this is not a on/off button, but a dropdown list with user group levels. The off value is just there to disable the feature. Same goes to the other fields with type accesslevel

**Comment by dgt41** [comment]
Ok synchronised!

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on dbb6124e75277c63912ab0636d7d3eb50366073c<br /><br />Still, it works OK :-D<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on dbe6ef95d7586898f14b574c8e5f84ea60b550b2<br /><br />And, Still, it works OK :-D<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by infograf768** [comment]
@joomla/cms-maintainers 
When this is merged, let TTs know about the new TinyMCE lang string (not the ini file).

@dgt41 
`<creationDate>2005-2015</creationDate>` => 2016

**Comment by wilsonge** [comment]
Thanks @dgt41 :)

**Comment by infograf768** [comment]
The new lang string is
"Insert\/Edit code sample":

**Comment by mikeh000** [comment]
Can it be version 4.3.3 that gets merged to Joomla 3.5? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8553">issues.joomla.org/joomla-cms/8553</a>.</sub>

**Comment by infograf768** [comment]
it is 4.3.3 which has been merged.

**Pull request merged**